packages/bits/src/lib/select-v2/base-select-v2.ts

```diff
@@ -318,11 +318,11 @@ export abstract class BaseSelectV2 implements AfterViewInit, AfterContentInit, C
     }
 
     protected getValueFromOptions(options = this.selectedOptions): OptionValueType | OptionValueType[] | null {
-        return this.multiselect ? options.map(o => o.value) : options[0]?.value || null;
+        return this.multiselect ? options.map(o => o.value) : options[0]?.value || "";
```

Comment:
What does this do?

Reply:
It checks to see if the value is in the list of options and if it is not it will set it to "" instead of null.  This gets ran anytime selectedOptions gets ran.